tools/reviewer/job/sync/LineNumberConverter.java

```diff
@@ -87,6 +87,11 @@ public int getLineNumber() {
     }
   }
 
+  @SuppressWarnings({
```

Comment:
Why do we suppress and not fix?

Reply:
@oferb same thing as [here](https://github.com/google/startup-os/pull/490#discussion_r244553836), will address in next PR